inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd

```diff
@@ -41,71 +48,18 @@ opts_chunk$set(
 
 ```{r metadata}
 metadata <- data.frame(
-  Reviewers = "{{{reviewers}}}",
-  Documentbeheerder = "{{{file_manager}}}",
-  Procedure = "{{{procedure}}}",
-  Revisie = "{{{revision}}}"
+  Reviewers = paste(params$reviewers, collapse = ", "),
+  File_manager = params$file_manager,
+  Procedure = params$procedure,
```

Comment:
@hansvancalster is 'procedure' a term needing explanation (with regard to other terms) within Readme?

Reply:
"procedure" is the term we used up till now (docx style protocols), but when I started transition to GitHub I have used protocol-code (in the Readme.md of protocols repo). So they are synonyms, but for consistency I will change all instances of procedure to protocol-code.

Reply:
Done in 4f365d9

Reply:
Thanks!